CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 Change log
 -----------
 
+* Various fixes [Theodor]
```

Comment:
Was thinking of that, but these fixes are not specific to the TX2, they just never surfaced before.